### DISCUSSION TITLE
Document links for presets config and defaults are broken

### DISCUSSION THREAD

**Discussion by adamlyka** [renovatebot/renovate#28969] [Request Help] [answered]
### What would you like help with?

Other

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

`Full Config Presets` link is broken: https://github.com/renovatebot/renovate/blob/main/docs/usage/config-presets.md#how-to-use-preset-configs

`Renovate's default presets` link is broken:
https://github.com/renovatebot/renovate/blob/main/docs/usage/key-concepts/presets.md#built-in-presets

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
These pages are dynamically built/generated based on source code and work when the pages are viewed on docs.renovatebot.com

**Reply by adamlyka** [reply]
Ah, this can be confusing when coming in from google search
